k6/src/modules/auth/login.js

```diff
@@ -0,0 +1,62 @@
+import http from 'k6/http';
+import { check } from 'k6';
+import User from './model/User.js';
+
+/**
+ * @param params
+ * @returns User
+ */
+export function loginUi(params) {
+    const res = http.request(
+        'POST',
+        params.url + '/tao/Main/login',
+        {
+            login: params.login,
+            password: params.password,
+            loginForm_sent: 1,
+            connect: 'Log in'
+        },
+        {
+            redirects: 999,
```

Comment:
Is it a good practice to use this?

Reply:
Put 0 and I am asserting only the 302 now. Login will run just one time in the setup method.